docs/Home.md

```diff
@@ -11,8 +11,8 @@ OAuth for Spring Security provides an [OAuth](http://oauth.net)
 implementation for
 [Spring Security](http://projects.spring.io/spring-security/).
 Support is provided for the implementation of OAuth providers and
-OAuth consumers. There is support for [Oauth 1(a)](oauth1.html) (including
```

Comment:
The `*.html` links are valid. If you go to the [main page](https://projects.spring.io/spring-security-oauth/docs/Home.html), you will see that the links work properly.